server/html.go

```diff
@@ -183,35 +234,20 @@ func (s *Server) NewInbox(w http.ResponseWriter, r *http.Request) {
 
 	err = s.db.SaveNewInbox(i)
 	if err != nil {
-		log.Printf("NewInbox: failed to save email: %v", err)
+		log.Printf("CreateRouteFromInbox: failed to save email: %v", err)
 		http.Error(w, "Failed to save new inbox", http.StatusInternalServerError)
 		return
 	}
 
 	sess.Values["id"] = i.ID
 	err = sess.Save(r, w)
 	if err != nil {
-		log.Printf("NewInbox: failed to set session cookie: %v", err)
+		log.Printf("CreateRouteFromInbox: failed to set session cookie: %v", err)
 		http.Error(w, "Failed to set session cookie", http.StatusInternalServerError)
 		return
 	}
 
-	io := indexOut{
-		Static:   s.getStaticDetails(),
-		Messages: nil,
-		Inbox:    i,
-		Expires: expires{
-			Hours:   "23",
-			Minutes: "59",
-		},
-	}
-
-	err = indexTemplate.ExecuteTemplate(w, "base", io)
-	if err != nil {
-		log.Printf("NewInbox: failed to write response: %v", err)
-		http.Error(w, "Failed to write response", http.StatusInternalServerError)
-		return
-	}
+	http.Redirect(w, r, "/", http.StatusFound)
```

Comment:
I think this may need to be after the lambda check and wait. Not sure whether it will wait otherwise.